moskito-core/src/main/java/net/anotheria/moskito/core/stats/impl/StatValueTypeUtility.java

```diff
@@ -50,28 +50,28 @@
 	/**
 	 * Factory instance for long values.
 	 */
-	private static LongValueHolderFactory longValueHolderFactory = new LongValueHolderFactory();
+	private static IValueHolderFactory longValueHolderFactory = new LongValueHolderFactory();
```

Comment:
Same as before: The more abstract type leads to a better encapsulation and allows to replace the implementation more easily.